src/libraries/System.Data.Odbc/src/Common/System/Data/ProviderBase/DbBuffer.cs

```diff
@@ -19,7 +19,7 @@ protected DbBuffer(int initialSize) : base(IntPtr.Zero, true)
             if (0 < initialSize)
             {
                 _bufferLength = initialSize;
-                RuntimeHelpers.PrepareConstrainedRegions();
```

Comment:
This project targets netstandard + netframework.
(Comment applies file-wide.)

Reply:
The file is in an ItemGroup that only targets NetCoreApp.